testool/Config.toml

```diff
@@ -10,6 +10,13 @@ path="tests/src/GeneralStateTestsFiller/**/*"
 max_gas = 0
 max_steps = 100000
 
+[[suite]]
```

Comment:
Not sure, do we need to add this @ChihChengLiang ?

Reply:
We should add this one since our TLOAD TSTORE is implemented.